plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/github.ts

```diff
@@ -67,6 +71,21 @@ export function createPublishGithubAction(options: {
             type: 'string',
             enum: ['private', 'public', 'internal'],
           },
+          collaborators: {
```

Comment:
Think this schema should be the same as the frontend one.

Something like this:

```json
{
  "collaborators": {
    "title": "Collaborators",
    "description": "Provide users with permissions",
    "type": "array",
    "items": {
      "type": "object",
      "required": [
        "username",
        "access"
      ],
      "properties": {
        "access": {
          "type": "string",
          "description": "The type of access for the user",
          "default": "pull",
          "enum": [
            "push",
            "pull",
            "admin",
            "maintain",
            "triage"
          ]
        },
        "username": {
          "type": "string",
          "description": "The username or group"
        }
      }
    }
  }
}
```

Reply:
Updated 👍